book.json

```diff
@@ -1,5 +1,8 @@
 {
     "gitbook": ">=3.2.0",
+    "variables": {
+      "django_version": "2.0.6"
```

Comment:
Awesome! just what I had in mind and I learnt something new about gitbook :-p

Thanks for the changes @Dayof

Reply:
Very cool! :+1:

Now we just need some magic to change everything else that needs to be changed when we upgrade the version again. :stuck_out_tongue:

 :wink: